src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 0.3s;
```

Comment:
0.3s meet two time, you can use variable better. But ok

Reply:
Agree with @tatyana22 , fix it

Reply:
also fix that